2024-09-01 16:02:37,921 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Configure stats pid to 39536
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Loading settings from /Users/gino/.config/wandb/settings
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Loading settings from /Users/gino/promisegat4/wandb/settings
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_init.py:_log_setup():524] Logging user logs to /Users/gino/promisegat4/wandb/run-20240901_160237-kjrkaa7y/logs/debug.log
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/gino/promisegat4/wandb/run-20240901_160237-kjrkaa7y/logs/debug-internal.log
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_init.py:init():607] calling init triggers
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-01 16:02:37,922 INFO    MainThread:39536 [wandb_init.py:init():632] re-initializing run, found existing run on stack: 1m9idjx8
2024-09-01 16:02:37,923 INFO    MainThread:39536 [wandb_run.py:_finish():2160] finishing run spycoderyt/promisegat4/1m9idjx8
2024-09-01 16:02:37,923 INFO    MainThread:39536 [jupyter.py:save_history():444] not saving jupyter history
2024-09-01 16:02:37,924 INFO    MainThread:39536 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-09-01 16:02:37,924 INFO    MainThread:39536 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-09-01 16:02:37,924 INFO    MainThread:39536 [wandb_run.py:_atexit_cleanup():2424] got exitcode: 0
2024-09-01 16:02:37,924 INFO    MainThread:39536 [wandb_run.py:_restore():2406] restore
2024-09-01 16:02:37,924 INFO    MainThread:39536 [wandb_run.py:_restore():2412] restore done
2024-09-01 16:02:37,924 INFO    MainThread:39536 [wandb_run.py:_on_finish():2677] communicating current version
2024-09-01 16:02:37,968 INFO    MainThread:39536 [wandb_run.py:_on_finish():2686] got version response 
